client/src/containers/Editor/Editor.js

```
@@ -128,16 +129,21 @@ class Editor extends Component {
 
     let error = null;
     if (this.state.loadingError) {
+      const message = this.state.loadingError.value;
       error = (
         <div className="editor__file-preview-message--file-missing">
-          {i18n._t('AssetAdmin.FILE_MISSING', 'File cannot be found')}
+          {(message.contains('Unexpected token < in JSON'))
```

Apparently this is necessary, but I find the output message hiding the underlying error (potential server error) a bit misleading. Is there no better solution here?